py/torch_tensorrt/_compile.py

```diff
@@ -44,6 +46,7 @@ class _ModuleType(Enum):
     nn = 0
     ts = 1
     fx = 2
+    ep = 3
```

Comment:
Unrelated to this change, but the docstring on this `Enum` is not relevant to the content